### PR TITLE
qemu.tests.netkvm_in_use: Update some commands for checking pci.

### PR DESCRIPTION
**Pull request by CongLi**
Signed-off-by: Cong Li <coli@redhat.com>

id: 1231126

### CODE REVIEW

**Comment by CongLi** [comment]
117951 16:36:20 INFO | PASS 1-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.W       in2012.x86_64.r2.io-github-autotest-qemu.netkvm_in_use.with_hotplug.before_bg_test

**Comment by CongLi** [comment]
@suqinhuang 

Could you help review this one?

Thanks.

File: qemu/tests/cfg/netkvm_in_use.cfg [hunk 1]
```diff
@@ -56,9 +56,9 @@
             suppress_exception = no
             pci_type = nic
             pci_model = virtio-net-pci
-            match_string = "Virtio network device"
-            reference_cmd = wmic nic list
-            find_pci_cmd = wmic nic list
+            match_string = "VirtIO Ethernet"
```
Comment (suqinhuang) [inline qemu/tests/cfg/netkvm_in_use.cfg]: is the key word same in different product, e.g rhel6, rhel7

Comment (CongLi) [inline qemu/tests/cfg/netkvm_in_use.cfg]: 22:59:46 INFO | PASS 1-smp_1.2048m.repeat1.run_test.Host_RHEL.m6.u8.qcow2.virtio_scsi.up.virtio_net.Win2016.x86_64.io-github-autotest-qemu.netkvm_in_use.with_hotplug.before_bg_test

**Comment by PyLearner** [comment]
PASS 1-smp_16.16384m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.netkvm_in_use.with_hotplug.before_bg_test

Acked-by: Meng Yang meyang@redhat.com

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Pull request merged**